src/mcts/search.h

```diff
@@ -222,6 +222,13 @@ class SearchWorker {
         this->RunTasks(i);
       });
     }
+    target_minibatch_size_ = params_.GetMiniBatchSize();
```

Comment:
Not sure whether it's better, but maybe make `params_.GetMiniBatchSize()` an `std::option<int>`?

Then it would be possible to do `target_minibatch_size_ = params_.GetMiniBatchSize().value_or(search_->network_->GetMiniBatchSize())`.

Also, why not put target_minibatch_size_ and max_out_of_order_ into `search_` rather than SearchWorker?

Reply:
The change to `std::optional<int>` seems a bit overcomplicated, even more since this way allows us to keep the 32 minibatch-size default for selfplay with no changes.
It seemed cleaner to put `target_minibatch_size_` and `max_out_of_order_` in SearchWorker since they are only used there.